packages/core/protocols/src/proto/dxos/halo/credentials.proto

```diff
@@ -173,6 +173,9 @@ message DeviceProfileDocument {
   optional string os = 5;
   optional string os_version = 6;
   optional DeviceType type = 7;
+  optional string hue = 8;
+  optional string emoji = 9;
+  optional string displayName = 10;
```

Comment:
This isn’t used anywhere yet, though if this is functionality we might like to have it’s worth keeping it.